src/main/resources/jenkins/branch/MultiBranchProjectWelcomeView/main.jelly

```diff
@@ -22,10 +22,21 @@
  ~ THE SOFTWARE.
  -->
 <?jelly escape-by-default='true'?>
-<st:compress xmlns:st="jelly:stapler">
-  <p>
-    This project scans branches in your SCM and generate a job for each of them,
-    but you have no branches configured.
-    <a href="./configure">Configure the project</a> and then <a href="./indexing">index branches</a> to get started.
-  </p>
+<st:compress xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+  <div style="margin: 20px 0; padding: 10px 30px 20px 30px; background: #f8f8f8; border-radius: 10px; max-width: 55em;">
+    <h1>This folder is empty</h1>
+  
+    <p>There are no branches found with a <code>Jenkinsfile</code>.</p>
+  
+    <p>
+      Jenkins automatically builds and manages projects in branches that contain a file named <code>Jenkinsfile</code> in the project root.
+      This file must contain a valid 
+      <a href="https://github.com/jenkinsci/workflow-plugin/blob/master/TUTORIAL.md#understanding-flow-scripts">Jenkins Workflow script</a>.
+    </p>
```

Comment:
I see. In that case you could simply override the welcome view in `WorkflowMultiBranchProject`, though it would be preferable to have `MultiBranchProjectWelcomeView` have the boilerplate page layout and then `st:include` a view on its `owner` for the details, with the default being the current content, and that being overridden in `WorkflowMultiBranchProject`.